--- rake-compiler-0.8.3/spec/lib/rake/extensiontask_spec.rb.newver	2013-02-27 23:12:42.000000000 +0900
+++ rake-compiler-0.8.3/spec/lib/rake/extensiontask_spec.rb	2013-02-27 23:14:20.000000000 +0900
@@ -471,6 +471,7 @@
       'rbconfig-1.9.1' => '/some/path/version/1.9.1/to/rbconfig.rb',
       'rbconfig-1.9.2' => '/some/path/version/1.9.1/to/rbconfig.rb',
       'rbconfig-1.9.3' => '/some/path/version/1.9.1/to/rbconfig.rb',
+      'rbconfig-2.0.0' => '/some/path/version/2.0.0/to/rbconfig.rb',
       'rbconfig-3.0.0' => '/some/fake/version/3.0.0/to/rbconfig.rb'
     }
   end
